core/lib/multivm/src/versions/vm_latest/mod.rs

```diff
@@ -23,7 +23,7 @@ pub(crate) use self::{
 pub(crate) mod bootloader;
 pub mod constants;
 mod implementation;
-mod old_vm;
+pub mod old_vm;
```

Comment:
why is it required? i'd strongly suggest not to make it public

Reply:
pub(crate) fn test_v28_precompiles_disabled<VM: TestedVm>() 

It was required for this test

What would be the best way to refactor?

Reply:
using `pub(crate)` instead of `pub` should be ok